recipe/bld.bat

```diff
@@ -1,9 +1,11 @@
 @echo off
 setlocal EnableDelayedExpansion
 
-if "%gpu_variant%"=="cuda-12" or "%gpu_variant%"=="cuda-11" (
+if "%gpu_variant:~0,5%"=="cuda-" (
     set CMAKE_ARGS=!CMAKE_ARGS! -DCMAKE_CUDA_ARCHITECTURES=all
-    set LLAMA_ARGS=!LLAMA_ARGS! -DLLAMA_CUBLAS=ON
+    set LLAMA_ARGS=!LLAMA_ARGS! -DLLAMA_CUDA=ON
```

Comment:
Would it be worth adding a comment to say that this enables cublas? I found it slightly confusing that we are disabling blas when the blas implementation is set to "cublas". I had to look at llama-cpp's CMakeLists to understand what was going on.

Reply:
done